cluster/images/kubemark/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM debian:jessie
+
+RUN apt-get update
+RUN apt-get install -y wget vim rsync ca-certificates
+RUN update-ca-certificates
+
+COPY kubemark.sh /kubemark.sh
+RUN chmod a+x /kubemark.sh
+
+COPY kubernetes-server-linux-amd64.tar.gz /tmp/kubemark.tar.gz
+RUN tar xzf /tmp/kubemark.tar.gz
+
+RUN cp /kubernetes/server/bin/hyperkube /
+RUN cp /kubernetes/server/bin/kubemark /
```

Comment:
Where does "kubemark" binary come from? (i.e. what is the source file and where is it getting built?)